tests/functional/kvs_many_pfx.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2021 Micron Technology, Inc. All rights reserved.
+
+from contextlib import ExitStack
+
+from hse2 import hse
+
+from utility import lifecycle, cli
+
+# creates 1million + prefixes , adds data to each prefix and validates.
+
+# Test params
+# total number of prefixes
+NUM_PFXS = 1024 * 1024 + 1024
+# number of keys per prefix
+NUM_KEYS = 10
+
+hse.init(cli.CONFIG)
+
+
+def create_pfx_data(kvdb, kvs, pfx):
+    pfx_bytes = pfx.to_bytes(4, byteorder="big")
+
+    with kvdb.transaction() as txn:
+        for sfx in range(NUM_KEYS):
+            key = pfx_bytes + sfx.to_bytes(1, byteorder="big")
+            kvs.put(key, key, txn=txn)
+
+
+def verify_pfx_data(kvdb, kvs, pfx):
+    pfx_bytes = pfx.to_bytes(4, byteorder="big")
```

Comment:
Does byteorder matter here? Can we not leave it up to the system default?

Reply:
the byte order is big so that int-order matches lexical-order.